src/olympia/devhub/helpers.py

```diff
@@ -225,3 +226,14 @@ def version_disabled(version):
     disabled = [status == amo.STATUS_DISABLED
                 for _id, status in version.statuses]
     return all(disabled)
+
+
+@register.function
+def pending_activity_log_count_for_developer(version):
+    alog = ActivityLog.objects.for_version(version).filter(
+        action__in=amo.LOG_REVIEW_QUEUE_DEVELOPER)
+
+    dev_replies = alog.filter(action=amo.LOG.DEVELOPER_REPLY_VERSION.id)
+    if not len(dev_replies) == 0:
+        alog = alog.filter(created__gt=dev_replies[0].created)
```

Comment:
Same here as in https://github.com/mozilla/addons-server/pull/3454/files#r78188491